jumpscale/sals/marketplace/apps_chatflow.py

```diff
@@ -74,9 +74,10 @@ def _choose_flavor(self, flavors=None):
 
     def _get_pool(self):
         self.currency = "TFT"
-        available_farms = []
         farm_names = [f.name for f in j.sals.zos.get()._explorer.farms.list()]
+        random.shuffle(farm_names)
```

Comment:
what about instead of shuffle, we sort the largest resources first to reduce the probability of no ssd storage in user pools, as in apps ?

Reply:
I think shuffling is better to avoid getting stuck with a mal-functioning farm for all deployments